paddle/phi/kernels/sparse/unary_kernel.h

```diff
@@ -225,5 +225,21 @@ SparseCsrTensor ReshapeCsr(const Context& dev_ctx,
   return csr;
 }
 
+template <typename T, typename Context>
+void SliceCooKernel(const Context& dev_ctx,
+                    const SparseCooTensor& x,
+                    const phi::IntArray& axes_arr,
+                    const phi::IntArray& starts_arr,
+                    const phi::IntArray& ends_arr,
```

Comment:
Why use parameter name of `axes_arr`, `starts_arr` and `ends_arr`? No other kernel are not named like this, just using `axes`, `starts` and `ends` is better?

Reply:
Thank you very much for your review! 

The reason is that the data type of these parameters is `IntArray`, and I will get their underlying data in implementing the function, like `axes_arr.GetData()`. For convenience, I use `axes_arr` as the parameter's name and `axes` as the variable name indicating the underlying data described above in the function body.

And I found that the names of the `starts` and `ends` parameters are also `starts_array` and `ends_array` in DenseTensor's slice function:

https://github.com/PaddlePaddle/Paddle/blob/develop/paddle/phi/kernels/impl/slice_kernel_impl.h#L102-L114

Reply:
Additionally, the configuration of DenseTensor's slice API in `legacy_ops.yaml` is:

https://github.com/PaddlePaddle/Paddle/blob/cdbf62f829a245bc36ca6db93d2f3c838a1ce64a/paddle/phi/api/yaml/legacy_ops.yaml#L906-L913

Reply:
The signature of the SliceKernel should mainly refers to https://github.com/PaddlePaddle/Paddle/blob/d1d43c22c81e12b825cfbadb304435deda0f2389/paddle/phi/kernels/slice_kernel.h#L25-L32

The function mentioned above in Paddle/paddle/phi/kernels/impl/slice_kernel_impl.h is its implementation, and it's better to be the same, If you are interested, can propose another PR to correct the parameter names in slice_kernel_impl.h

Reply:
Thanks for your reply! I will propose a PR to correct it when I have free time.